lib/preview-features.ts

```diff
@@ -40,8 +40,7 @@ export const previewFeatures: PreviewFeature[] = [
     title: 'Expense AI assistant',
     description: 'Introducing an AI assistant to help you create expenses.',
     publicBeta: false,
-    closedBetaAccessFor: ['opencollective', 'opensource', 'foundation', 'europe', 'design', 'engineering'],
+    closedBetaAccessFor: ['opencollective', 'opensource', 'foundation', 'europe'],
```

Comment:
Should we leave engineering and design on? I mean, either way, anyone can submit expenses to the hosts...

Reply:
It's to simplify the check in the API; there we can simply check if the user is an admin of an internal host